components/autogen/src/format-pages.txt

```diff
@@ -1732,6 +1732,22 @@ presenceRating = Fair
 utilityRating = Good
 reader = SISReader
 
+[OMERO Pyramid]
+developer = `Open Microscopy Environment <http://www.openmicroscopy.org/>`_
+bsd = yes
+versions = 1.0.0
+weHave = * a :doc:`specification document </developers/omero-pyramid>`\n
+* several OMERO Pyramid datasets\n
+* the ability to produce additional datasets
+pixelsRating = Outstanding
```

Comment:
Regarding the "outstanding" rating, what's the justification for this?

- I understand it can't represent floating point for pixels?
- It supports almost zero metadata
- It's really only useful in the context of OMERO

So would it make sense to use some more realistic assessment of the pixels, metadata and utility?